pkg/core/bundle_list_test.go

```diff
@@ -0,0 +1,289 @@
+package core
```

Comment:
Why the different formatting?

Reply:
Eh.. generated..

Reply:
@kerneltime could please you elaborate a bit? I don't understand

Reply:
tab space here seems at 8? vs 3

Reply:
@kerneltime there are no spaces to indent in gofmt golang source, only tabs.

Reply:
@fredbi i opened up the file in IDE and the formatting looks fine but here on GitHub the tab width seems to be set at 8..

Reply:
Ok